docs/virtual_machines/host-devices.md

```diff
@@ -176,3 +176,93 @@ spec:
     name: cloudinitdisk
 ```
 
+## USB Host Passthrough
+
+Since KubeVirt v1.1, we can provide USB devices that are plugged in a Node to the VM running in the
+same Node.
+
+### Requirements
+
+Cluster admin privilege to edit the KubeVirt CR in order to:
+
+- Enable the `HostDevices` [feature gate](../operations/activating_feature_gates.md)
+- Edit the `permittedHostDevices` configuration to expose node USB devices to the cluster
+
+### Exposing USB Devices
+
+In order to assign USB devices to your VMI, you'll need to expose those devices to the cluster under
+a resource name. The device allowlist can be edited in KubeVirt CR under
+`configuration.permittedHostDevices.usb`.
+
+For this example, we will use the `kubevirt.io/storage` resource name for the device with `vendor:
+"46f4"` and `product: "0001"` [^1].
+
+
+```yaml
+spec:
+  configuration:
+    permittedHostDevices:
+      usb:
+        - resourceName: kubevirt.io/storage
+          selectors:
+            - vendor: "46f4"
+              product: "0001"
+```
+
+After adding the `usb` configuration under `permittedHostDevices` to the KubeVirt CR, KubeVirt's
+device-plugin will expose this resource name and you can use it in your VMI.
+
+
+### Adding USB to your VM
+
+Now, in the VMI configuration, you can add the `devices.hostDevices.deviceName` and reference the
+`resource name` provided in the previous step, and also give it a local name, for example:
```

Comment:
```suggestion
resource name provided in the previous step, and also give it a local `name`, for example:
```
Sorry, looks like I missed this earlier. I don't think we should mark up 'resource name' as that's not the option name, but I think we should mark up the local 'name' so that it's easier to identify in the subsequent code block.